plugins/out_stackdriver/stackdriver.c

```diff
@@ -1458,7 +1551,15 @@ static int stackdriver_format(struct flb_config *config,
         msgpack_pack_str_body(&mp_pck,
                               ctx->pod_name, flb_sds_len(ctx->pod_name));
     }
-
+    else {
```

Comment:
I think we need to decide whether `logging.googleapis.com/monitored_resource` or the the resource in output plugin has a higher priority to set the monitored resource, and I lean towards making `logging.googleapis.com/monitored_resource` as higher priority because it is set per payload.

Let's discuss this part offline with other team members.

After we have the decision, we should add testcases about the priority between output plugin's configuration & `logging.googleapis.com/monitored_resource`.

Reply:
Discussed offline, logging.googleapis.com/monitored_resource has the highest priority. Done.